src/org/etools/j1939_84/controllers/part01/Part01Step03Controller.java

```diff
@@ -110,6 +135,18 @@ protected void run() throws Throwable {
             // All the values should be the same
             addWarning("6.1.3.3.a - An ECU responded with a value for OBD Compliance that was not identical to other ECUs");
         }
+
+        // 6.1.3.3.b Warn if any response received from a non-OBD ECU provides OBD Compliance values of 0, FBh, FCh,
+        // FDh, FEh, or FFh.
+        parsedPackets.stream().forEach(p -> {
+            if (!p.isObd() && p.getOBDCompliance() != 0x5) {
+                addWarning(String.format("6.1.3.3.b Response received from a non-OBD ECU %s provides OBD Compliance values of %Xh.",
+                                         p.getModuleName(),
+                                         p.getOBDCompliance()));
+            }
+        });
+
+        // 6.1.3.3.a Info for DM5 replies from non-OBD ECUs.
```

Comment:
Um, 6.1.3.3.b & 6.1.3.3.c appear to be missing.  Did you forget to check those in or am I missing something?

Reply:
6.1.3.3.b is there, but has imilar issues.
What does 6.1.3.3.c mean?